roles/remove_stale_lun/tasks/remove_mpath_device.yml

```diff
@@ -0,0 +1,30 @@
+- name: Get underlying disks (paths) for a multipath device and turn them into a list.
+  ansible.builtin.shell: dmsetup deps -o devname "{{ lun_wwid }}" | cut -f 2 |cut -c 3- |tr -d "()"
+  register: disks
+  delegate_to: "{{ host_item.address }}"
+  connection: ssh
+  check_mode: "no"
+  with_items:
+    - "{{ host_info.ovirt_hosts }}"
+  loop_control:
+    loop_var: host_item
+
+- name: Remove from multipath device.
+  ansible.builtin.shell: multipath -f "{{ lun_wwid }}"
+  delegate_to: "{{ host_item.address }}"
+  connection: ssh
+  check_mode: "no"
+  with_items:
+    - "{{ host_info.ovirt_hosts }}"
+  loop_control:
+    loop_var: host_item
+
+- name: Remove each path from the SCSI subsystem.
```

Comment:
For removal of path, you can use module `file` with `state: absent`

Reply:
It actually doesn't remove file from the system, it writes `1` into `/sys/block/$dev/device/delete` which cuases remove of the iSCSI path/device from the system.

Reply:
ACK